packages/mdc-textfield/README.md

```diff
@@ -53,9 +46,9 @@ npm install @material/textfield
 </div>
 ```
 
-> NOTE: Text field supports `text`, `number`, and `password` input types (e.g., `<input type="password" class="mdc-text-field__input">`).
+> NOTE: Text field supports `text` and `password` input types (e.g., `<input type="password" class="mdc-text-field__input">`).
 >
-> Other input types (such as `date`) are not currently supported.
+> Other input types (such as `number` and `date`) are not currently supported.
```

Comment:
Nice catch!

Reply:
I actually think this is also from the bad sync. I am reverting it back to how it was.